lib/components/Modal.js

```diff
@@ -47,24 +56,36 @@ var Modal = React.createClass({
       portalClassName: 'ReactModalPortal',
       ariaHideApp: true,
       closeTimeoutMS: 0,
-      shouldCloseOnOverlayClick: true
+      shouldCloseOnOverlayClick: true,
+      parentSelector: document.body
```

Comment:
I think maybe specifying a function here would be best something like `() => document.body` would probably do.